src/main/java/com/rocketden/main/controller/v1/GameController.java

```diff
@@ -0,0 +1,40 @@
+package com.rocketden.main.controller.v1;
+
+import com.rocketden.main.dto.room.GetRoomRequest;
+import com.rocketden.main.dto.room.RoomDto;
+import com.rocketden.main.service.RoomService;
+import com.rocketden.main.dto.game.StartGameRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GameController extends BaseRestController {
+    
+    private final RoomService service;
+
+    @Autowired
+    public GameController(RoomService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/start")
+    public ResponseEntity<Void> startGame(@RequestBody StartGameRequest request) {
```

Comment:
This should return the `RoomDto` I believe; it's not used now, but it will for sure need to be used in the future as the frontend will need to know what users are in the room and such.

If you didn't add this because you didn't want to add something in that wouldn't be used, then I'm open for discussion (also thoughts @alankbi on this?), but otherwise, I think this should return the appropriate `RoomDto`, as well as updating that on the frontend (it wouldn't have to use it at all, perhaps just pass the details into the `/game` url.

The more I type this, the more unsure I get as you already have a lot added here, so just let me know if you think it should be added or not.

Reply:
I'm also not sure/don't have a preference; for consistency, RoomDto makes sense, but for minimizing data transfer, Void is also reasonable, so either way seems reasonable to me

Reply:
Sure, I can have it return a RoomDto and the front end doesn't do anything with it until we need that information.